Error rebuilding spark images

Might be a transient. If not, need to make sure ca-certificates-java gets installed.

```
The following packages have unmet dependencies:
 openjdk-8-jre-headless : Depends: ca-certificates-java but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
Removing intermediate container 22cb4d5b0965
The command '/bin/sh -c echo 'deb http://cdn-fastly.deb.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/jessie-backports.list &&     apt-get -y update &&     apt-get install -y --no-install-recommends openjdk-8-jre-headless &&     rm /etc/apt/sources.list.d/jessie-backports.list &&     apt-get clean &&     rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
make: *** [build/pyspark-notebook] Error 100
```

--- a/pyspark-notebook/Dockerfile
+++ b/pyspark-notebook/Dockerfile
@@ -13,7 +13,7 @@ ENV HADOOP_VERSION 2.7
 # Temporarily add jessie backports to get openjdk 8, but then remove that source
 RUN echo 'deb http://cdn-fastly.deb.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/jessie-backports.list && \
     apt-get -y update && \
-    apt-get install -y --no-install-recommends openjdk-8-jre-headless && \
+    apt-get install --no-install-recommends -t jessie-backports -y openjdk-8-jre-headless ca-certificates-java && \
     rm /etc/apt/sources.list.d/jessie-backports.list && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*

